### PR TITLE
localize settings header and footer

### PR DESCRIPTION
**Pull request by 6a68**
## Because

- we want to localize the header and footer

## This pull request

- localizes the header and footer :-)

- Note: I feel a little weird about putting the strings for the footer (which lives in fxa-react) over in the fxa-settings App component's ftl file. This is in keeping with the approach from #7327 and I can't think of a better approach, but other suggestions welcome!

- [Update: regular casing restored.] ~~Also note: I've used SpOnGeBoB cAsInG to convince myself that the l10n transforms are actually getting applied. But nobody else is doing that in the other l10n commits, so I can revert to normal casing if that's preferred.~~ 

## Issue that this pull request solves

Closes: #5058

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [x] I have added necessary documentation (if appropriate).
- [x] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

**Comment by LZoog** [comment]
> Also note: I've used SpOnGeBoB cAsInG to convince myself that the l10n transforms are actually getting applied. But nobody else is doing that in the other l10n commits, so I can revert to normal casing if that's preferred.

FWIW I vaguely remember @meandavejustice doing this in Payments (I think?) a while back for the same reason, and @clouserw pushing back on the off chance something fails and an end-user sees the weird casing.

**Comment by 6a68** [comment]
Cool, thanks @LZoog! I think Dave and I both picked up that habit from m&m when we were working on lockbox

File: packages/fxa-settings/public/locales/en-US/settings.ftl [hunk 1]
```diff
@@ -7,6 +7,23 @@
...
+app-footer-privacy-notice = Website Privacy Notice
+app-footer-terms-of-service = Terms of Service
+
+# HeaderLockup component
+
+header-menu-open = Close menu
+header-menu-closed = Site navigation menu
+header-back-to-top-link =
+  .title = Back to top
+header-title = Firefox accounts
```
Comment (meandavejustice) [inline packages/fxa-settings/public/locales/en-US/settings.ftl:22]: This should be singular `Firefox account` according to figma

**Review by meandavejustice — CHANGES_REQUESTED** [review]
* One comment on translation copy.
* need to remove `packages/fxa-settings/package-lock.json`

**Comment by 6a68** [comment]
Thanks for the review! Updated, re-pushed, ready for another look

**Review by chenba — APPROVED** [review]
WFM

**Pull request merged**